spec/requests/api/v1/docs/display_ads_spec.rb

```diff
@@ -68,6 +68,8 @@
             placement_area: { type: :string, enum: DisplayAd::ALLOWED_PLACEMENT_AREAS,
                               description: "Identifies which area of site layout the ad can appear in" },
             tag_list: { type: :string, description: "Tags on which this ad can be displayed (blank is all/any tags)" },
+            article_exclude_ids: { type: :string,
+                                   description: "Articles this ad should *not* appear (blank is all/any article), comma-separated list of integer Article IDs" }, # rubocop:disable Layout/LineLength
```

Comment:
```suggestion
                                   description: "Articles this ad should *not* appear (blank means no articles are disallowed, and this ad can appear next to any/all articles). Comma-separated list of integer Article IDs" }, # rubocop:disable Layout/LineLength
```

I'm not totally sure mine is better, but something about the way the above is worded has the potential to be confusing. Or maybe it's just me?